scripts/install-darwin-multi-user.sh

```diff
@@ -4,7 +4,17 @@ set -eu
 set -o pipefail
 
 # System specific settings
+# Notes:
+# - up to macOS Big Sur we used the same GID/UIDs as Linux (30000:30001-32)
+# - we changed UID to 301 because Big Sur updates failed into recovery mode
+#   we're targeting the 200-400 UID range for role users mentioned in the
+#   usage note for sysadminctl
+# - we changed UID to 331 because Sequoia now uses UIDs 300-304 for its own
+#   daemon users
+# - we changed GID to 331 alongside above just because it hides the nixbld
+#   group from the Users & Groups settings panel :)
 export NIX_FIRST_BUILD_UID="${NIX_FIRST_BUILD_UID:-331}"
+export NIX_BUILD_GROUP_ID="${NIX_BUILD_GROUP_ID:-331}"
```

Comment:
This might be silly, but – is there any chance of this group being associated with `_nixbld1` specifically because of sharing the GID with it? If so, we could avoid that by picking 330 instead. I guess that per‐user groups sharing the IDs of their corresponding users is just a convention so hopefully nothing in the system is going to *assume* it and cause some kind of weird behaviour down the road, but all this system role ID stuff makes me superstitious enough to worry about it.

Reply:
I hope not, but that's definitely in the class of risk that makes me a bit cautious about changing it without a reason (but I think you've provided a sufficiently-good reason to try it). 

Hopefully we'll have some degree of confirmation from people running this for at least a few days before merge.

Reply:
Should we pick 330 instead just in case? That matches the staus quo on Linux (30000 ≠ 30001), the previous status quo on Darwin (30000 ≠ 301), and doesn’t seem to have any additional risks.

I’m overdue for reinstalling my crusty old Nix setup anyway, so I will probably try this out soon as long as I can get the Darwin sandbox actually fixed.